#from one single satellite
uint32 sat         # satellite ID
float64[] CN0       # carrier-to-noise density ratio (signal strength) [dB-Hz]
float64 psr         # pseudorange without ionosphere/troposphere error
float64 mp          # multipath
float64 ecefX       # satellite's position in ECEF
float64 ecefY
float64 ecefZ
